superset/dao/base.py

```diff
@@ -50,14 +50,17 @@ class BaseDAO:
 
     @classmethod
     def find_by_id(
-        cls, model_id: Union[str, int], session: Session = None
+        cls,
+        model_id: Union[str, int],
+        session: Session = None,
+        no_filter: bool = False,
```

Comment:
nit: `no_filter` feels slightly ambiguous. Would `skip_filter` or `ignore_filter` be more explicit?

Reply:
Agreed, let's go with `skip_filter`